Test Controller/device_test.py

```
@@ -34,146 +33,159 @@ def OnMidiIn(event):
+def play_harmony_arpeggio():
+    """화성을 아르페지오로 재생"""
+    global current_harmony_notes, harmony_channel
+    
+    if not current_harmony_notes:
+        return
+    
+    channels.selectOneChannel(harmony_channel)
+    
+    # 아르페지오 재생 (순차적으로)
+    def play_arpeggio():
+        for i, note in enumerate(current_harmony_notes):
```

Ruff (0.11.9)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)